### DISCUSSION TITLE
Slider cannot be set to zero

### DISCUSSION THREAD

**Discussion by windpoiuytr** [katspaugh/wavesurfer.js#3504] [Q&A]
The left side of the slider can never be zero
![微信图片_20240119202105](https://github.com/katspaugh/wavesurfer.js/assets/100547177/e71d69f2-2a26-40cd-8d35-369943d81ce1)
![微信图片_20240119202110](https://github.com/katspaugh/wavesurfer.js/assets/100547177/85df748a-e382-4647-a6e3-d256d0be642d)

**Comment by katspaugh** [comment]
I've just tested it on https://wavesurfer.xyz/examples/?5f22003081e3a845afdc68e8bb18657a and I could easily drag or resize the region to start (almost) at 0.

**Reply by windpoiuytr** [reply]
The longer the audio time, the greater the error.
